src/base/utils/net.h

```diff
@@ -28,13 +28,23 @@
 
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
+
 class QString;
 
 namespace Utils
 {
     namespace Net
     {
         bool isValidIP(const QString &ip);
+
+        template <typename T>
+        T unquote(const T &str)
```

Comment:
And if you have not noticed, it is better to use the word "unquote**d**" since we don't modify source string but create new one (with unquoted source in it).